dipy/reconst/brainsuite_shore.py

```diff
@@ -0,0 +1,668 @@
+from __future__ import division
+import warnings
```

Comment:
in general, it is better to split the import into three visual groups
1. build-in packages
2, third party packages (external)
3. import from this package (internal)